projects/canopy/src/lib/forms/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './checkbox-group/index';
```

Comment:
Yes, it makes sense to be in a separate commit, will do that.

Reply:
Thanks, it makes me think about how we should document issues like this. Because there's a previous commit/release where the export was missing, and then this new commit/rekease which fixes it. 

I wonder if we need to add something to the release notes where it was missing, or create a issue to capture what it was, which version(s) was affected and then closed and marked as fixed with the new version 🤔 

Maybe something to chat about when everyone is back from holiday.

Reply:
Its definitely a good point to raise for discussions.